### ISSUE TITLE
Homepage redesign: Add anchor to Get In Touch section

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3023]
### Overview
As a developer, I want to be able to go directly to the Get In Touch section on the redesigned homepage so that I can quickly and easily go to that section without scrolling. For this issue, we will add an anchor to the Get In Touch section.  

### Details
<details>
  <summary><h4>Screenshot of Get in Touch Section on the Redesigned Homepage</h4></summary>
  <img width="949" alt="get-in-touch" src="https://user-images.githubusercontent.com/31293603/161402137-288d4358-4ba6-4df3-99ca-9008c12ce4f2.png">
</details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch` and the pull request created from this issue should be pushed into that feature branch. Please read and follow the steps in the [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch) wiki article.
- [x] Before line 1 in _includes/forms/contact-us.html, add `<a class="anchor" id="get-in-touch"></a>` so that
  ```
  <section class="home--contact home--section">
  ``` 
  becomes
  ```
  <a class="anchor" id="get-in-touch"></a>
  <section class="home--contact home--section">
  ```
- [x] Using Docker, check that going to `localhost:4000/#get-in-touch` jumps to the Get In Touch section on the homepage
- [ ] Once the pull request associated with this issue is approved and merged, 
  - [x] Please update and edit issue https://github.com/hackforla/website/issues/2393 by
    - [x] Checking off the dependency for this issue
    - [x] If all dependencies are checked off, please move issue https://github.com/hackforla/website/issues/2393 to the New Issue Approval column and remove the Dependency label
  - [x] Please update and edit issue #2455 by
    - [x] Checking off the dependency for this issue
    - [x] If all dependencies are checked off, please move issue #2455 to the New Issue Approval column and remove the Dependency label 

### Resources/Instructions
- https://github.com/hackforla/website/blob/467a797f7c9518fcb466b6bdd034de27050b838e/_includes/forms/contact-us.html#L1
- Example page that has anchors: https://www.hackforla.org/about
- Tracked in https://github.com/hackforla/website/issues/2481#issuecomment-968349924
- Related to issues
  - https://github.com/hackforla/website/issues/2393
  - https://github.com/hackforla/website/issues/2455

**Comment by Providence-o** [comment]
@KazushiR are you no longer working on this issue? I can see that you assigned yourself to it, moved it from the prioritized backlog to inprogress and then removed it from the board entirely and you're still assigned to it. 

Additionally on another issue #3016 , it looks like you removed it from the prioritized backlog and the project board. 

Also in #2961, you assigned yourself, added a milestone and then unassigned yourself from the issue 

Please explain what you're trying to accomplish in both issues so that we can help you. 

@JessicaLucindaCheng @SAUMILDHANKAR

**Comment by KazushiR** [comment]
> @KazushiR are you no longer working on this issue? I can see that you assigned yourself to it, moved it from the prioritized backlog to inprogress and then removed it from the board entirely and you're still assigned to it.
> 
> Additionally on another issue #3016 , it looks like you removed it from the prioritized backlog and the project board.
> 
> Also in #2961, you assigned yourself, added a milestone and then unassigned yourself from the issue
> 
> Please explain what you're trying to accomplish in both issues so that we can help you.
> 
> @JessicaLucindaCheng @SAUMILDHANKAR

Apologies. I was trying to assign myself to an issue that I could work on. I will be working on this issue. Apologies on the confusion.

Availability: 4/15/22 10AM - 2PM
ETA: 4 Hours

**Comment by JessicaLucindaCheng** [comment]
@KazushiR At the time you self-assign an issue, please move the issue from the "Prioritized Backlog" to the "In Progress" column. For detailed instructions, see [2.4.b Move this issue from the ‘Prioritized Backlog’ to the ‘In progress’ & back](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#24b-move-this-issue-from-the-prioritized-backlog-to-the-in-progress--back) in the CONTRIBUTING.md.

**Comment by SAUMILDHANKAR** [comment]
@KazushiR Just wanted to remind you that a medium issue should be picked up once your small issue has been merged. Please see the following link for the reasons of this progression: https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#21-hack-for-la-contributor-expectations. Therefore, requesting you to please pick up a medium issue next once this one has been merged. Also, please move the issue to In-progress column as suggested by Jessica in a comment above.
We can go over how to pick up an issue and further steps in our next team meeting break-out room if required. Please let me know.

**Comment by SAUMILDHANKAR** [comment]
@KazushiR As per our last discussion, unassigning you from this issue so you could pick up a medium issue next which would be more in line with your issue progression. But if you feel like you have already done substantial amount of work for this issue and would like to finish it before moving on to the next issue, you can pick up this issue again instead and work on it. I will leave it up to you, but please work on only one issue at a time. Thank you.

**Comment by JessicaLucindaCheng** [comment]
@geedtd 
At the time you self-assign an issue, make sure to write in a comment the following:
- Availability: "How much time will you have this week to work on this issue?"
- ETA: "When do you expect this issue to be completed?"

**Comment by kathrynsilvaconway** [comment]
@geedtd  Hi, I am unassigning you from this because we haven't heard from you in quite a while. If you would still like to be a part of the HFLA dev team, you can reassign yourself and update your status.

**Comment by CalvinTan607** [comment]
i. Availability: 8/9 - 8/10
ETA: By the end of 8/9
